hack/golang/golangci-lint.yaml

```diff
@@ -112,6 +112,11 @@ issues:
   # excluded by default patterns execute `golangci-lint run --help`
   exclude: []
 
+  exclude-rules:
```

Comment:
had to disable 'dupl' as it was complaining on the DRYed code, as it becomes too much similar to each other

Reply:
This will disable this linter for all code in this repo. Can we make this a `nolint` annotation on each file? I know its a bit of a PITA, but this linter _can_ be useful sometimes in helping find places where we can refactor. You can disable the linter for the whole file with:

```
// nolint: dupl
package X
```